WebRoot/skins/harmony/skin.css

```diff
@@ -70,6 +70,10 @@
 	background-color:@AppC@;
 }
 
+.ZmConvView2FullWidth {
+	width: 100%;
```

Comment:
In the chrome/safari its setting width automatically for firefox need to setup width.